src/app/modules/user/user-projects/user-projects.component.ts

```diff
@@ -182,13 +211,13 @@ export class UserProjectsComponent implements OnInit {
       this.modalRef.setClass('project-modal');
```

Comment:
Take a look at [this helper class](https://github.com/DigitalExcellence/dex-frontend/blob/develop/src/app/utils/project-detail-modal.util.ts)

Reply:
Hi Walter, I'm wondering why I have to look at the helper class of the modal, I added pagination to the userprojects page and did not touch any of the modals

Reply:
Seems like the userProjects page is opening the modals itself, it should be using the help class instead. Would be nice if you could implement that in this PR too :) Or if @MeesvanStraten/ @DaveBouman wants to make it a different issue that is cool too